spec/controllers/review_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe ReviewsController do
+  describe 'POST create' do
+
+    let(:video) { Fabricate(:video) }
+    
+    context 'with authentication' do
+    
+      let(:user) { Fabricate(:user) }
+      
+      before do
+        session[:user_id] = user.id
+      end
+
+      context 'with valid input' do
+
+        before do
+          post :create, review: Fabricate.attributes_for(:review), video_id: video.id
+        end
+        
+        it 'should sets @video' do
+          expect(Video.first).to eq(video)
```

Comment:
Okay, I am apparently tired. I apologize. I didn't realize that this spec was testing the creation of a review, not a video. Ignore my last comment...the last part of it is still valid though.